apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -136,20 +136,6 @@ func (api *UserManagerAPI) addOneUser(ctx context.Context, arg params.AddUser) p
 	var activationKey []byte
 	var err error
 
-	// TODO(anvial): Legacy block to delete when user domain wire up is complete.
-	if arg.Password != "" {
-		_, err = api.state.AddUser(arg.Username, arg.DisplayName, arg.Password, api.apiUserTag.Id())
```

Comment:
I believe this allows the deletion of this method from state, but that can follow.

Reply:
In an upcoming PR :)